grid2op/Observation/_ObsEnv.py

```diff
@@ -244,7 +238,6 @@ def _reset_to_orig_state(self):
             if save:
                 import pandapower as pp
                 pp.to_json(self.backend._grid, "test_action1.json")
-        # print("load after applying act: {}".format(self.backend._grid.load["p_mw"].values[0]))
```

Comment:
If it possible to cleanup the code block above ? Should be :
```
self._backend_action_set.all_changed()
self._backend_action = copy.deepcopy(self._backend_action_set)
```
Without conditional branchind and json files export